.github/workflows/conformance-ingress.yaml

```diff
@@ -243,7 +243,21 @@ jobs:
         timeout-minutes: 5
         run: |
           lb=$(kubectl get ingress basic-ingress -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
-          curl -s -v --connect-timeout 5 --max-time 20 --retry 3 --fail -- http://"$lb"
+          
```

Comment:
Instead of adding this logic, have you tried using cURL's `--retry-all-errors` flag? There is also `--retry-connrefused`.

It could simplify things here.

Reply:
Awesome, didn't know about this flag. Works according to CI, so going with this! Thx